tests/Oryx.Integration.Tests/LocalDockerTests/PythonEndToEndTests.cs

```diff
@@ -560,6 +560,46 @@ public async Task CanBuildAndRun_Tweeter3App()
                 });
         }
 
+        [Theory]
+        [InlineData("2.7")]
+        [InlineData("3.6")]
+        [InlineData("3.7")]
+        public async Task CanBuildAndRun_ShapelyFlaskApp(string pythonVersion)
+        {
+            // Arrange
+            var appName = "shapely-flask-app";
+            var hostDir = Path.Combine(_hostSamplesDir, "python", appName);
+            var volume = DockerVolume.Create(hostDir);
+            var appDir = volume.ContainerDir;
+            var portMapping = $"{HostPort}:{ContainerPort}";
+            var script = new ShellScriptBuilder()
+                .AddCommand($"cd {appDir}")
+                .AddCommand($"oryx -appPath {appDir} -bindPort {ContainerPort}")
+                .AddCommand(DefaultStartupFilePath)
+                .ToString();
+            var imageVersion = "oryxdevms/python-" + pythonVersion;
+
+            await EndToEndTestHelper.BuildRunAndAssertAppAsync(
+                appName,
+                _output,
+                volume,
+                "oryx",
+                new[] { "build", appDir },
```

Comment:
FYI, this does not use virtual environment as that is not the default yet. I think ideally we should use that as AppService uses that.

Reply:
sure